addons/mail/static/src/components/message/message.xml

```diff
@@ -19,7 +19,7 @@
             }" role="group" t-att-aria-label="messageView and messageView.message.messageTypeText" t-on-click="_onClick" t-on-mouseenter="state.isHovered = true" t-on-mouseleave="state.isHovered = false" t-att-data-message-local-id="messageView and messageView.message.localId"
         >
             <t t-if="messageView" name="rootCondition">
-                <MessageInReplyToView t-if="messageView.messageInReplyToView" messageInReplyToViewLocalId="messageView.messageInReplyToView.localId"/>
+                <MessageInReplyToView t-if="messageView.messageInReplyToView and threadView and !threadView.__values.threadViewer.chatWindow" messageInReplyToViewLocalId="messageView.messageInReplyToView.localId"/>
```

Comment:
`__values` is internal code of discuss models, and should never be used by business code.
`x.__values.y` => `x.y`.

FYI, relational fields store their local ids in `__values` and models get the model record when using the getter, using the local id internally thanks to `__values`. `x.__values.y.z` does not work when `y` is a relational field, because `__values.y` is a local id (string) and `__values.y.z` is `undefined`.